### ISSUE TITLE
Set title tag based on page metadata

### ISSUE THREAD

**Issue by dylans** [dojo/site#170]
Currently every page on the Dojo website has the same title tag, so everything just says dojo.io.

We should infer a page title from the metadata we have about each page.

**Comment by matt-gadd** [comment]
@dylans good catch, I think this could potentially be a good issue on dojo/framework for the router to support perhaps?

**Comment by KaneFreeman** [comment]
Dependent on https://github.com/dojo/framework/issues/601 (PR: https://github.com/dojo/framework/pull/620)
